apps/balancer-tools/src/app/apr/pool/(components)/HistoricalCharts/index.tsx

```diff
@@ -0,0 +1,84 @@
+"use client";
+
+import { Suspense, useState } from "react";
+
+import { PoolStatsData } from "#/app/apr/api/route";
+import { Spinner } from "#/components/Spinner";
+import { Tabs } from "#/components/Tabs";
+import { trimTrailingValues } from "#/lib/utils";
+
+import HistoricalAPRChart from "./HistoricalAPR/HistoricalAPRChart";
+import HistoricalSwapFeeChart from "./HistoricalSwapFee/HistoricalSwapFeeChart";
+
+export default function HistoricalCharts({
+  poolId,
+  roundId,
+}: {
+  poolId: string;
+  roundId?: string;
+}) {
+  const charts = ["APR", "Swap Fee"];
+
+  const [selectedTab, setSelectedTab] = useState(charts[0]);
+
+  function handleTabClick(event: React.FormEvent<HTMLButtonElement>) {
+    const target = event.target as HTMLButtonElement;
+    setSelectedTab(target.innerText);
+  }
+  return (
+    <div className="border border-blue6 bg-blue3 rounded p-4 w-full">
+      <Tabs defaultValue={charts[0]} value={selectedTab}>
+        <Tabs.ItemTriggerWrapper>
+          {charts.map((chartType) => (
+            <Tabs.ItemTrigger
+              tabName={chartType}
+              key={chartType}
+              triggerBgColor="bg-blue6"
+              onClick={handleTabClick}
```

Comment:
why add a layer of indirection? 😅 
```suggestion
              onClick={() => setSelectedTab(chartType))}
```